From cb2cdcbb875d20da394027a8da210b3b05fa9aeb Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 26 Jul 2022 14:59:33 +0200
Subject: [PATCH] mpc85xx: create modules.mk

This is done similarly as for Turris Omnia in mvebu.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 target/linux/mpc85xx/modules.mk | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 target/linux/mpc85xx/modules.mk

diff --git a/target/linux/mpc85xx/modules.mk b/target/linux/mpc85xx/modules.mk
new file mode 100644
index 0000000000..7bb5704f73
--- /dev/null
+++ b/target/linux/mpc85xx/modules.mk
@@ -0,0 +1,14 @@
+define KernelPackage/leds-turris-1x
+   SUBMENU:=$(LEDS_MENU)
+   TITLE:=LED support for CZ.NIC's Turris 1.x
+   DEPENDS:=@TARGET_mpc85xx
+   KCONFIG:=CONFIG_LEDS_TURRIS_1X
+   FILES:=$(LINUX_DIR)/drivers/leds/leds-turris-1x.ko
+   AUTOLOAD:=$(call AutoLoad,60,leds-turris-1x,1)
+endef
+
+define KernelPackage/leds-turris-1x/description
+   This option enables support for the LEDs of the CZ.NIC Turris 1.x routers.
+endef
+
+$(eval $(call KernelPackage,leds-turris-1x))
\ No newline at end of file
-- 
2.34.1

